modules/light-clients/09-localhost/light_client_module.go

```diff
@@ -12,16 +11,23 @@ import (
 	"github.com/cosmos/ibc-go/v8/modules/light-clients/09-localhost/internal/keeper"
 )
 
+var _ exported.LightClientModule = (*LightClientModule)(nil)
+
 type LightClientModule struct {
-	keeper keeper.Keeper
+	keeper        keeper.Keeper
+	storeProvider exported.ClientStoreProvider
```

Comment:
localhost we should actually just provide the full ibc store. I don't think it needs a store provider